.travis.yml

```diff
@@ -16,5 +16,7 @@ matrix:
       env: TOXENV=flake8
     - python: 2.7
       env: TOXENV=ansible-lint
+    - python: 2.7
+      env: TOXENV=ansible2-lint
```

Comment:
Based on my understanding, If I take this approach, I can't trigger ansible 2.x testing without an alternative invocation (e.g. `ANSIBLE_VERSION=">=2.0" tox -e ansible-lint`, which is generally breaking people expectations of interacting with tox locally (or mine, at least).

I think I would favor separate environments solely to keep calling environments consistent. I may have missed something here that gets around that, though.